bindings/nodejs/lib/utils/utils.ts

```diff
@@ -130,12 +130,15 @@ export class Utils {
      * @param rent Rent cost of objects which take node resources.
```

Comment:
Something like, feel free to edit
```suggestion
     * @param storageScoreParameters Storage score of objects which take node resources.
```

Reply:
oops